packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -203,11 +211,22 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
             app.use(json({ limit: '100mb', extended: true }))
             app.use(urlencoded({ limit: '100mb', extended: true }))
 
+            // Remove \r and \n to prevent header injection attacks
+            app.use((req, res, next) => {
```

Comment:
Please add the name. It’s easier to debug by traces

Reply:
Done